src/app/shared/post-it-collections/collection-name/post-it-collection-name.component.ts

```diff
@@ -32,8 +32,9 @@ export class PostItCollectionNameComponent {
 
   @Input() public collection: CollectionModel;
 
-  @Output()
-  public changed = new EventEmitter<string>();
+  @Output() public changed = new EventEmitter<string>();
+  @Output() public selected = new EventEmitter();
+  @Output() public unselected = new EventEmitter();
```

Comment:
Cannot the last two be merged into a single output with `boolean` parameter? We should always try to reduce the number of interactions between components.

The other thing is that JS events are usually called `change`, `select`, etc. But it is not really important.